routes/api/comments.js

```diff
@@ -0,0 +1,65 @@
+const createError = require("http-errors");
+const express = require("express");
+
+const { asyncHandler } = require("../utils");
+const { CommentLike } = require("../../db/models");
+const { restoreUser } = require("../../auth");
+
+const router = express.Router();
+
+/* Like a comment */
+router.post(
+  "/comments/:id/likes",
+  restoreUser,
+  asyncHandler(async (req, res, next) => {
+    if (!res.locals.authenticated) {
+      /* Return error 401, if
+        user is not authenticated, or
+        user is not liker */
+      res.status(401);
+    } else if (res.locals.authenticated) {
+      /* Create comment like */
+      const likedComment = await CommentLike.create({
+        userId: res.locals.user.id,
+        commentId: req.params.id,
+      });
+
+      /* Set status 201, and
+        respond with json, likedComment */
+      res.status(201).json(likedComment);
+    }
+  })
+);
+
+/* Unlike a comment */
+router.delete(
+  "/comments/likes/:id",
+  restoreUser,
+  asyncHandler(async (req, res, next) => {
+    /* Initialize like object */
+    const like = await CommentLike.findByPk(req.params.id);
+
+    /* Return error 404, if 
+      there is no like object */
+    if (!like) {
+      res.status(404);
```

Comment:
Utilize error handlers for error status codes. See previous examples you've implemented in this branch.

Reply:
Utilized error handler